oag/ja-jp/content/topics/releasenotes/oag/oag-release-notes.htm

```diff
@@ -0,0 +1,354 @@
+<!DOCTYPE html>
```

Comment:
Drat, this is the older version, from a previous template. Will need to update this here and on the LL branch.